Doc/whatsnew/3.13.rst

```diff
@@ -433,7 +433,7 @@ New Features
   :c:expr:`const char * const *`, :c:expr:`const char **` or
   :c:expr:`char * const *` in C++ and :c:expr:`char * const *` in C
   without explicit type cast.
-  This can be overridden by defining the macro :c:macro:`PY_CXX_CONST`
+  This can be overridden by defining the macro ``PY_CXX_CONST``
```

Comment:
Can we use this?

```suggestion
  This can be overridden by defining the macro :c:macro:`!PY_CXX_CONST`
```

Reply:
To note, while they both silence the warning, they have different semantics in the source and rendering in the output.

However, with the changes suggested above, it is a moot point as you can link this and also simplify this What's New entry by referring readers to the macro description, which explains this (instead of duplicating the explanation here).